packages/react-native-gesture-handler/android/build.gradle

```diff
@@ -158,9 +158,16 @@ android {
                     arguments "-DREACT_NATIVE_DIR=${REACT_NATIVE_DIR}",
                             "-DREACT_NATIVE_MINOR_VERSION=${REACT_NATIVE_MINOR_VERSION}",
                             "-DANDROID_STL=c++_shared",
-                            "-DANDROID_SUPPORT_FLEXIBLE_PAGE_SIZES=ON"
+                            "-DANDROID_SUPPORT_FLEXIBLE_PAGE_SIZES=ON",
+                            // Turn off build IDs for reproducibility (ensuring the same code will produce the same bundle when built)
+                            // See https://gitlab.com/IzzyOnDroid/repo/-/wikis/Reproducible-Builds/RB-Hints-for-Developers#no-funny-build-time-generated-ids
+                            // for more information
+                            "-DCMAKE_SHARED_LINKER_FLAGS=-Wl,--build-id=none"
                     abiFilters(*reactNativeArchitectures())
                 }
+                ndkBuild {
+                    arguments "APP_LDFLAGS+=-Wl,--build-id=none"
+                }
```

Comment:
RNGH doesn't use `ndk` (only `cmake`) so it seems like those lines are irrelevant?

Reply:
Yes, I'll get back to it later 😅 Thanks for pointing this out ❤️